modules/SDL.lua

```diff
@@ -81,4 +126,6 @@ function SDL:DeleteFile()
   end
 end
 
+setAllSdlBuildOptions()
```

Comment:
@aderiabin, This function is executed as many times as module SDL "required"
For now this module is required in testbase and connecttest (dummy_connect_test, hmi_values etc.)

So need some logic to execute this function only once